doc/source/whatsnew/v1.1.0.rst

```diff
@@ -114,7 +114,7 @@ Missing
 
 MultiIndex
 ^^^^^^^^^^
-
+- Bug in :meth:`Dataframe.loc` when used with a :class:`MultiIndex`. The returned values were not in the same order as the given inputs (:issue:`22797`)
```

Comment:
can you make a mini-example (separate sub-section). I think this is very hard to visualize from the text, but the basic example we are using is very clear.

Reply:
Sure thing.